pkg/filters/operator.go

```diff
@@ -0,0 +1,12 @@
+package filters
+
+const (
+	Equals            string = "="
+	In                string = "in"
```

Comment:
In the description i see an example with == for In query. Please update with the latest working one.